src/main/resources/standardTestCases/final_call_logs_dir_local.test

```diff
@@ -0,0 +1,21 @@
+# Tests that a runtime option to save call logs is copying said logs appropriately.
+
+name: final_call_logs_dir_local
+backends: [Local, Tes]
+backendsMode: "any"
+tags: [localdockertest]
```

Comment:
What's the net effect of all these options?  It can run in "any" of `[Local, Tes]`, as long as it's `Local`?